parachain/primitives/router/src/lib.rs

```diff
@@ -37,6 +43,25 @@ pub trait ConvertMessage {
 	fn convert(origin: H160, dest: u32, payload: Payload) -> (MultiLocation, Xcm<()>);
 }
 
+#[derive(Clone, Eq, PartialEq, Encode)]
+pub enum StatemineCall {
+	#[codec(index = 53u8)]
+	Assets(AssetsCall),
+}
+
+pub type Signature = MultiSignature;
+type AccountId = <<Signature as Verify>::Signer as IdentifyAccount>::AccountId;
+type AccountIndex = ();
+type Address = MultiAddress<AccountId, AccountIndex>;
```

Comment:
None of this is necessary. An AccountId is usually configured to be 32 byte array: `[u8; 32]`.